### DISCUSSION TITLE
Ceph Cluster Version Nautilus (>=15.0.0) ?

### DISCUSSION THREAD

**Discussion by mathrock74** [ceph/ceph-csi#3736] [Q&A] [answered]
Hi,

what does "Ceph Cluster Version Nautilus (>=15.0.0)" in Support Matrix exactly mean? Ceph Nautilus is 14 according to https://docs.ceph.com/en/latest/releases/index.html.

thx

**Comment by Madhu-1** [comment]
The above metrics are what ceph version is supported with cephcsi for a specific feature.

**Reply by mathrock74** [reply]
Ok, and what ceph version is now exactly required: 14 (=Nautilus) or 15 (>=15.0.0)?

**Reply by Madhu-1** [reply]
it  depends on the feature mentioned here https://github.com/ceph/ceph-csi#support-matrix

**Reply by mathrock74** [reply]
Ok, let's say feature "Dynamically provision, de-provision Block mode RWO volume" for RBD needs "Ceph Cluster Version Nautilus (>=15.0.0)". I am confused because "Nautilus" means Ceph 14 and "(>=15.0.0)" means Ceph 15, at least to me. What's so hard to understand about this question?

**Reply by Madhu-1** [reply, marked as answer]
I see the problem. It's a documentation bug. It should be Octopus, not Nautilus.

**Reply by mathrock74** [reply]
Thank you
